src/Compilers/Core/Portable/Diagnostic/Diagnostic.cs

```diff
@@ -502,7 +502,7 @@ internal static int GetDefaultWarningLevel(DiagnosticSeverity severity)
         /// </summary>
         internal virtual bool IsNotConfigurable()
         {
-            return AnalyzerManager.HasNotConfigurableTag(this.CustomTags);
+            return DefaultCompilationWithAnalyzersHost.HasNotConfigurableTag(this.CustomTags);
```

Comment:
>DefaultCompilationWithAnalyzersHost.HasNotConfigurableTag(this.CustomTags); [](start = 19, length = 75)

Do not share this implementation here.  Diagnostic should not have any code tied to implementation of analyzers.